src/main/java/org/truffleruby/core/CoreLibrary.java

```diff
@@ -1447,6 +1447,7 @@ public DynamicObject getTruffleKernelOperationsModule() {
             "/core/support.rb",
             "/core/string.rb",
             "/core/random.rb",
+            "/core/truffle/kernel_operations.rb", // Needed for thread to define hooked variables.
```

Comment:
Is there something special about this comment. All the files are in a order based on what is needed when.